### PR TITLE
Media print for back of card items

### PR DESCRIPTION
**Pull request by mwernimont**
@isuftin See any LESS functionality I could use in this?

### CODE REVIEW

File: coastal-hazards-portal/src/main/webapp/css/back/print.css [hunk 1]
```diff
@@ -1,192 +1,131 @@
...
+}
+#print-options {
+  background: #f5f5f5;
+  border: 1px solid #dddddd;
+  min-height: 50px;
+  margin: 10px 0;
+  border-radius: 5px;
+  padding: 5px 10px;
+}
+#print-options h3 {
```
Comment (isuftin) [inline coastal-hazards-portal/src/main/webapp/css/back/print.css]: You can nest here

    #print-options {
        font-family: blablahblah

        & h3 {
            margin-bottom : 5px
        }

        & input {
            margin: 2px 0 10px 0;
            width: 15px;
            height: 15px;
        }
    }

File: coastal-hazards-portal/src/main/webapp/css/back/print.css [hunk 2]
```diff
@@ -1,192 +1,131 @@
...
+#print-content {
+  min-height: 400px;
+  padding: 0px;
+}
+header {
+  height: 56px;
+  background: #003366;
+  color: #fff;
+}
+header h1,
```
Comment (isuftin) [inline coastal-hazards-portal/src/main/webapp/css/back/print.css]: More nesting here

File: coastal-hazards-portal/src/main/webapp/css/back/print.css [hunk 3]
```diff
@@ -1,192 +1,131 @@
...
+  height: 155px;
+  margin: -5px 0 10px 0;
+}
+.info-container {
+  width: 100%;
+  min-height: 50px;
+  line-height: 1.5em;
+  margin-bottom: 10px;
+}
+.info-container h3 {
```
Comment (isuftin) [inline coastal-hazards-portal/src/main/webapp/css/back/print.css]: More nesting

File: coastal-hazards-portal/src/main/webapp/css/back/print.css [hunk 4]
```diff
@@ -1,192 +1,131 @@
...
+  text-decoration: none;
+  font-weight: bold;
+}
+#site-url {
+  background: #003366;
+  color: #fff;
+  padding: 5px;
+  font-size: .9em;
+}
+@media screen and (min-width: 493px) {
```
Comment (isuftin) [inline coastal-hazards-portal/src/main/webapp/css/back/print.css]: Media can be nested as well

http://lesscss.org/features/#extend-feature-scoping-extend-inside-media

File: coastal-hazards-portal/src/main/webapp/css/back/print.css [hunk 5]
```diff
@@ -1,192 +1,131 @@
...
+}
+.info-container h3 {
+  font-size: 1.3em;
+  margin-bottom: 5px;
+}
+.info-container ul {
+  margin-left: 17px;
+}
+.info-container a {
+  color: #003366;
```
Comment (isuftin) [inline coastal-hazards-portal/src/main/webapp/css/back/print.css]: Bring duplicate colors into a variable at the top

Comment (isuftin) [inline coastal-hazards-portal/src/main/webapp/css/back/print.css]: Never mind, looks like you did

**Comment by mwernimont** [comment]
@isuftin Nesting is in place, but for some reason this media query nesting isn't clicking with me

**Comment by isuftin** [comment]
Take a look at what I did here: https://github.com/USGS-CIDA/coastal-hazards/pull/1164

**Comment by mwernimont** [comment]
aaahhh, now I see it

**Comment by mwernimont** [comment]
@isuftin Hows that look?

**Comment by isuftin** [comment]
:+1:

**Pull request merged**